frontend/src/book/store.js

```diff
@@ -132,17 +132,43 @@ export const setPageIndex = (pageIndex) => {
 
 export const movePage = (delta) => {
   setPageIndex(currentPage().index + delta);
+  if (delta < 0) {
+    removeStepsAnimations();
+  }
 };
 
+const removeStepsAnimations = () => {
+  const numOfSteps = currentPage().steps.length;
+  for (let i = 1; i <= numOfSteps; i++) {
+    const stepDiv = document.getElementById(`step-text-${i}`);
+    if (!stepDiv) {
+      continue;
+    }
+    stepDiv.style.animation = '';
+  }
+}
+
 export const moveStep = (delta) => {
   const stepIndex = currentStep().index + delta;
   const step = currentPage().steps[stepIndex];
   if (!step) {
     return;
   }
+
+  if (delta > 0) {
+    animateStep(stepIndex);
+  }
   setUserStateAndDatabase(["pagesProgress", localState.user.pageSlug, "step_name"], step.name);
 };
 
+const animateStep = (stepIndex) => {
+  const stepDiv = document.getElementById(`step-text-${stepIndex}`);
+  if (!stepDiv) {
+    return;
+  }
```

Comment:
I think this is no longer needed thanks to the new system

Reply:
Removed